grype/pkg/syft_json_provider.go

```diff
@@ -160,10 +185,6 @@ func (p *partialSyftPackage) UnmarshalJSON(b []byte) error {
 			PomGroupID:    group,
 			ManifestName:  name,
 		}
-	case "":
```

Comment:
was this part of the issue as well?

Reply:
Yes — I forget the exact execution path that was causing trouble, but I can dig that up if you'd like